joplin/base/blocks.py

```diff
@@ -1,6 +1,21 @@
+from wagtail.wagtailcore import blocks
 from wagtail.wagtailsnippets.blocks import SnippetChooserBlock
 
 
 class SnippetChooserBlockWithAPIGoodness(SnippetChooserBlock):
     def get_api_representation(self, model_instance, context=None):
         return model_instance.serializable_data()
+
+
+class WhatDoIDoWithBlock(blocks.StaticBlock):
+    class Meta:
+        icon = 'bin'
```

Comment:
interesting...

Reply:
haha, i thought because it was related to ARR a trash can makes sense. We can sub font awesome or similar. Here's the available icons http://kave.github.io/general/2015/12/06/wagtail-streamfield-icons.html